README.md

```diff
@@ -1,70 +1,29 @@
-[![Build Status](https://travis-ci.org/maxibor/coproID.svg?branch=master)](https://travis-ci.org/maxibor/coproID)   [![Documentation Status](https://readthedocs.org/projects/coproid/badge/?version=latest)](https://coproid.readthedocs.io/en/latest/?badge=latest) [![License: GPL v3](https://img.shields.io/badge/License-GPL%20v3-blue.svg)](https://www.gnu.org/licenses/gpl-3.0) [![DOI](https://zenodo.org/badge/148763195.svg)](https://zenodo.org/badge/latestdoi/148763195)
+# ![logo](docs/source/\_static/img/coproid_logo_small.jpg)  
```

Comment:
Your logo is hilarious but it would be nice to add an `nf-core` one too :smile:

Reply:
Logo moved with #7 
The coproID logo contains the nf-core logo, should I still add the nf-core logo also ?

![coproid_logo_small](https://user-images.githubusercontent.com/8630837/56805095-37766f80-6828-11e9-8529-3396ead51a71.jpg)

Reply:
I think its fine like this 👍 Adds a bit of humour!